tests/cart_test.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from src.shop.cart import Cart
+from src.shop.real_item import RealItem
+from src.shop.virtual_item import VirtualItem
+
+@pytest.fixture
+def valid_cart_data():
+    cart_name = "TestCart"
+    return Cart(cart_name)
+
+@pytest.mark.critical_workflow_smoke
+def test_add_real_item(valid_cart_data):
+    real_item = RealItem(name = "Toy", price = 100.1, weight = 2.0)
+    valid_cart_data.add_real_item(real_item)
+    assert valid_cart_data.total == (real_item .price + real_item .price * Cart.TAX + Cart.DELIVERY_FEE * real_item .weight)
```

Comment:
good practice to add messages for assertion

Reply:
code style - no spaces for call to instance attr

Reply:
ok, it is done